modules/about-workbench-images.adoc

```diff
@@ -4,7 +4,7 @@
 = About workbench images
 
 [role="_abstract"]
-A workbench image (sometimes referred to as a notebook image) is optimized with the tools and libraries that you need for model development. You can use the provided workbench images or an {productname-short} administrator can create custom workbench images adapted to your needs.
+A workbench image is optimized with the tools and libraries that you need for model development. You can use the provided workbench images or an {productname-short} administrator can create custom workbench images adapted to your needs.
```

Comment:
this _optimized_ really irks me

@coderabbitai please provide expert opinion

Reply:
agree with "pre-installed".